### ISSUE TITLE
Sales

### ISSUE THREAD

**Issue by Luciana478** [Luciana478/Dizar-Mobile#2]
How about the sales of our product? How much is it? The people should like? And the most critics? We need to make an acceptable work! Where is the community? This is what i called **Marketing!**

It's a good idea **try to get costumers** that _are_ interested on Dizar or **maintain it just for us?**

**Comment by RafaelCanarei5** [comment]
Yes, it's a good idea. Just be aware of the price: it can be about some 20 or 15, but not more that this!

**Comment by Luciana478** [comment]
This is good, because, even it's a low price, we should get money!

**Comment by Taiane34** [comment]
This can be difficult 'cause we don't have all recources for make a game engine!
